pkg/scheduler/scheduler.go

```diff
@@ -685,7 +685,9 @@ func (s *Scheduler) requeueAndUpdate(ctx context.Context, e entry) {
 
 	if e.status == notNominated || e.status == skipped {
 		patch := workload.AdmissionStatusPatch(e.Obj, true)
-		if workload.UnsetQuotaReservationWithCondition(patch, "Pending", e.inadmissibleMsg) {
+		change1 := workload.UnsetQuotaReservationWithCondition(patch, "Pending", e.inadmissibleMsg)
+		change2 := workload.PropagateResourceRequests(patch, &e.Info)
```

Comment:
```suggestion
		reservationIsChanged := workload.UnsetQuotaReservationWithCondition(patch, "Pending", e.inadmissibleMsg)
		resourceRequestsIsChanged := workload.PropagateResourceRequests(patch, &e.Info)
```

Could we avoid the magic variables?

Reply:
Changed variable names per suggestion.